addons/hr_holidays/static/src/views/view_dialog/form_view_dialog.xml

```diff
@@ -9,11 +9,11 @@
         <xpath expr="//button[contains(@class, 'o_form_button_cancel')]" position="after">
             <button class="btn btn-secondary" t-if="canDelete" t-on-click="deleteRecord" data-hotkey="x">Delete</button>
         </xpath>
-        <xpath expr="//div[contains(@class, 'o_cp_buttons')]" position="inside">
-            <div class="flex-grow-1 d-flex flex-row-reverse gap-1">
-                <button class="btn btn-secondary" t-if="canRefuse" t-on-click="() => this.onClick('action_refuse')" data-hotkey="z">Refuse</button>
-                <button class="btn btn-secondary" t-if="canApprove" t-on-click="() => this.onClick('action_approve')" data-hotkey="g">Approve</button>
+        <xpath expr="//div" position="inside">
+            <div class="ms-auto d-flex gap-1">
                 <button class="btn btn-secondary" t-if="canValidate" t-on-click="() => this.onClick('action_validate')" data-hotkey="v">Validate</button>
+                <button class="btn btn-secondary" t-if="canApprove" t-on-click="() => this.onClick('action_approve')" data-hotkey="g">Approve</button>
+                <button class="btn btn-secondary" t-if="canRefuse" t-on-click="() => this.onClick('action_refuse')" data-hotkey="z">Refuse</button>
```

Comment:
What's the rationale behind the order change here?

Reply:
still a wip but it is due to the removal of flex-row-reverse